### DISCUSSION TITLE
home assistant os e broadlink

### DISCUSSION THREAD

**Discussion by efestopini** [indomus/forum#2832] [HUB: Home Assistant]
Buongiorno, vorrei integrare un broadlik rm mini 3, ma sicuramente sbaglio qualcosa e non capisco cosa, avrò ripetuto la procedura 30 volte, ma mi si crea solo una entità "remote.mome che ho impostato", non dovrebbero essere 3 le entità ?

**Comment by indomus** [comment]
https://indomus.it/guide/integrare-attuatori-a-infrarossi-radiofrequenza-broadlink-a-home-assistant/

**Reply by efestopini** [reply]
grazie mille per il link ma ho già seguito quella guida, ma ho comunque solo una entità, elimino il rm mini3 da home assistant, cerco l' integrazione broadlink, aggiungo l'ip del dispositivo e compare una sola entità, stesso problema dell'utente luigi64 a qui è stato risposto cancella l'integrazione e rifalla, sta sbagliando qualcosa. Ho appunto ripetuto tutta la procedura almeno 30 volte sempre con lo stesso esito, non capisco cosa potrei sbagliare, la cosa mi sembra abbastanza semplice, chiede solo l'ip del dispositivo

**Reply by Scialla** [reply]
perché 3?
Una sola entità viene creata per ogni broadlink, e due servizi

**Reply by efestopini** [reply]
Io non ho i due servizi

**Reply by Scialla** [reply]
Come li cerchi?

**Reply by Scialla** [reply]
vai su strumenti sviluppatori, servizi, inserisci `remote.learn_command` oppure `remote.send_command` vedrai che li hai in elenco; scegli poi come entità il tuo brodalink e invoca il servizio
Vedrai che ci sono..

**Reply by efestopini** [reply]
grazie forse ho capito
